Formula/aom.rb

```diff
@@ -0,0 +1,58 @@
+class Aom < Formula
+  desc "Codec library for encoding and decoding AV1 video streams"
+  homepage "https://aomedia.googlesource.com/aom"
+  url "https://aomedia.googlesource.com/aom", :using => :git, :tag => "v1.0.0", :revision => "cb108fb20fec79291588f4925499ac4cb3e64bfd"
+
+  depends_on "cmake" => :build
+  depends_on "perl" => :build
+  depends_on "yasm" => :build
+
+  depends_on "ffmpeg" => :test
+
+  def install
+    args = %W[
+      -DCMAKE_INSTALL_PREFIX=#{prefix}
+      -DENABLE_EXAMPLES=on
+      -DENABLE_DOCS=off
+      -DENABLE_TESTDATA=off
+      -DENABLE_TESTS=off
+      -DENABLE_TOOLS=off
+    ]
+
+    mkdir "macbuild" do
+      system "cmake", "..", *args
+      system "make", "install"
+    end
+  end
+
+  test do
+    create_args = %w[
+      -f
+      lavfi
+      -i
+      color=black:352x288
+      -frames:v
+      5
+      blank.y4m
+    ]
+
+    encode_args = %w[
+      --webm
+      --tile-columns=2
+      --tile-rows=2
+      --threads=8
+      --cpu-used=8
+      --output=blank.webm
+      blank.y4m
+    ]
+
+    decode_args = %w[
+      --output=blank_decode.y4m
+      blank.webm
+    ]
+
+    system "ffmpeg", *create_args
```

Comment:
Rather than depend on `ffmpeg` to create this, could we download an example/test file from a known web location? (as a resource)

Reply:
Unfortunately, the smallest files I could find had at least 150 frames and the codec is very slow. It resulted in a timeout (the test took too long) when I tried to use one of the videos from https://media.xiph.org/video/derf/ instead.

Reply:
Found a smaller version of a publicly available video file that can be used to test encoding/decoding.